scripts/deploy-registry.template.ts

```diff
@@ -37,19 +37,70 @@ async function main() {
     const managerSelectors = getSelectors(managerFacet);
     // console.log("manager address:", managerFacet.address);
 
-    const registry = await ethers.getContractFactory('SubnetRegistry', { signer: deployer, libraries: {
+
+
+
+    //deploy subnet registry diamond
+    const registry = await ethers.getContractFactory('SubnetRegistryDiamond', { signer: deployer });
+
+    const registryConstructorParams = {
+      gateway: gatewayAddress,
+      getterFacet: getterFacet.address,
+      managerFacet: managerFacet.address,
+      subnetGetterSelectors: getterSelectors,
+      subnetManagerSelectors: managerSelectors,
+    };
+
+ 
+
+const facetCuts = [] //TODO
```

Comment:
This is a typo, right? I see that you are indeed pushing the new facet cuts here.
```suggestion
const facetCuts = []
```